core/lib/spree/core/controller_helpers/current_host.rb

```diff
@@ -7,7 +7,7 @@ module CurrentHost
         extend ActiveSupport::Concern
 
         included do
-          if Gem::Requirement.new("> 5.2").satisfied_by?(Rails.gem_version)
+          if Gem::Requirement.new(">= 6").satisfied_by?(Rails.gem_version)
```

Comment:
Can you explain why you're doing this?

Reply:
The remove deprecation warning on Rails 7. Might be better suitable for standalone PR now that I think of it. So we have 1 PR focussed only on adding Rails 7 compatibility.

Reply:
I think it's pertinent to have it on this PR. Besides, if there's a warning, CI will fail.

Reply:
ah.. thanks! Let's keep it then.

Reply:
Hey @peterberkenbosch, what was the deprecation warning?

Reply:
host is already set or something like that. I'm happy to remove all non-essential changes here actually.

Reply:
Just wanted to understand if that was coming from Solidus or Rails, because if it's the latter and we will receive warnings, the CI won't fail, it only fails for Solidus deprecations. cc @waiting-for-dev